Makefile

```diff
@@ -133,7 +140,7 @@ test/addons/.buildstamp: $(ADDONS_BINDING_GYPS) | test/addons/.docbuildstamp
 build-addons: $(NODE_EXE) test/addons/.buildstamp
```

Comment:
I didn't touch that yet because `$(NODE)` is either a release build or a debug build, whereas `$(NODE_EXE)` is always a release build.

The add-on tests all expect the shared object in out/Release but node-gyp puts the add-on in out/Debug when it detects a debug build.  I decided to put off fixing the tests until it was clear if this is the direction we're going to take.